infrastructure/code/code_html.go

```diff
@@ -68,11 +108,91 @@ func getDetailsHtml(issue snyk.Issue) string {
 
 	dataFlowHtml := getDataFlowHtml(additionalData)
 
+	// Header
 	html := replaceVariableInHtml(detailsHtmlTemplate, "issueId", issue.ID)
 	html = replaceVariableInHtml(html, "issueTitle", additionalData.Title)
+	html = replaceVariableInHtml(html, "issueType", getIssueType(additionalData))
 	html = replaceVariableInHtml(html, "severityText", issue.Severity.String())
-	html = replaceVariableInHtml(html, "dataFlow", dataFlowHtml)
+	html = replaceVariableInHtml(html, "severityIcon", getSeverityIconSvg(issue))
+
+	html = replaceVariableInHtml(html, "issueOverview", additionalData.Message)
+	// Data flow
 	html = replaceVariableInHtml(html, "dataFlowCount", fmt.Sprintf("%d", len(additionalData.DataFlow)))
+	html = replaceVariableInHtml(html, "dataFlow", dataFlowHtml)
+
+	// External example fixes
+	html = replaceVariableInHtml(html, "repoCount", fmt.Sprintf("%d", additionalData.RepoDatasetSize))
+	html = replaceVariableInHtml(html, "exampleCount", fmt.Sprintf("%d", len(additionalData.ExampleCommitFixes)))
+	html = replaceVariableInHtml(html, "tabsNav", getTabsHtml(additionalData.ExampleCommitFixes))
 
 	return html
 }
+
+func getFileName(filePath string) string {
+	if filePath == "" {
+		return ""
+	}
+
+	// Windows paths
+	filePath = strings.Replace(filePath, "\\", "/", -1)
+
+	// Handle trailing slash
+	if strings.HasSuffix(filePath, "/") {
+		return ""
+	}
+
+	filename := filepath.Base(filePath)
+	return filename
+}
+
+func getIssueType(additionalData snyk.CodeIssueData) string {
+	if additionalData.IsSecurityType {
+		return "Vulnerability"
+	}
+	return "Quality Issue"
+}
+
+func getRepoName(commitURL string) string {
+	trimmedURL := strings.TrimPrefix(commitURL, "https://")
+
+	re := regexp.MustCompile(`/commit/.*`)
+	shortURL := re.ReplaceAllString(trimmedURL, "")
+
+	tabTitle := shortURL
+	if strings.HasPrefix(shortURL, "github.com/") {
+		tabTitle = strings.TrimPrefix(shortURL, "github.com/")
+	}
+
+	if len(tabTitle) > 50 {
+		tabTitle = tabTitle[:50] + "..."
+	}
+
+	return tabTitle
+}
+
+func getSeverityIconSvg(issue snyk.Issue) string {
+	switch issue.Severity {
```

Comment:
I don't think this is an issue but maybe Design would know best? It renders in IntelliJ so technically I don't see why we wouldn't do this